ouroboros-network/src/Ouroboros/Network/PeerSelection/Simple.hs

```diff
@@ -50,26 +48,23 @@ withPeerSelectionActions
   -- (only if local root peers where non-empty).
   -> IO a
 withPeerSelectionActions localRootTracer publicRootTracer timeout readTargets _staticLocalRootPeers
-  localRootPeers publicRootPeers peerStateActions reqLedgerPeers getLedgerPeers k = do
-    localRootsVar <- newTVarIO Map.empty
+  localRootPeersVar publicRootPeersVar peerStateActions reqLedgerPeers getLedgerPeers k = do
+    localRootsVar <- newTVarIO []
     let peerSelectionActions = PeerSelectionActions {
             readPeerSelectionTargets = readTargets,
             readLocalRootPeers = pure [],
```

Comment:
See if we can pull changes from `p2p-master` branch that fix `readLocalRootPeers`.

Reply:
This is done in the commit: `p2p-governor: new root peers configuration (#3079)`.